### PR TITLE
validate CHEEPEPE

### PR DESCRIPTION
**Pull request by cheepepe**
# Validate [CHEEPEPE](https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg)

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/cheepepe_sol/status/1784062333511516552
- Coingecko/ CMC URL (If available): Still working to get Coingecko set up for CHEEPEPE

## Validation (Please check off boxes):
* [x] The metadata provided in the PR matches what is on-chain (Mandatory)
* [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
 * [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
* [x] My change matches the format in the file (no spaces between fields).
* [x] My token is already live and trading on Jupiter.
* [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.
* [x] Please make sure your pull request title has your token name. If it just says "Main", or "Validate", it will automatically be closed. PRs containing broken attestation or solscan links will also be closed.

### CODE REVIEW

**Comment by cheepepe** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

> * [x] Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
> * [x] Symbol: CHEEPEPE
> * [x] Token Decimals: 6
> * [x] Token icon: https://www.cheepepe.vip/token.png
> * [x] On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
> * [x] No duplication on Jupiter's strict list

**Comment by qqpmew** [comment]
I can hereby attest that the provided information are accurate, and the token is valid.

> * [x] Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
> * [x] Symbol: CHEEPEPE
> * [x] Token Decimals: 6
> * [x] Token icon: https://www.cheepepe.vip/token.png
> * [x] On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
 > * [x] No duplication on Jupiter's strict list

**Comment by Crussty93** [comment]
I can confirm that the information here is accurate and the token is valid and credible.
- Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- Symbol: CHEEPEPE
- Token Decimals: 6
- Token icon: https://www.cheepepe.vip/token.png
- On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- No duplication on Jupiter's strict list

**Comment by Manuella169** [comment]
Great project @cheepepe will moon

**Comment by Suhail51998** [comment]
Wow. First dog then cat and now mouse to the moon 🔥

**Comment by cottonmdalton** [comment]
I can hereby attest that the provided information is accurate and the token is valid.
- Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- Symbol: CHEEPEPE
- Token Decimals: 6
- Token icon: cheepepe.vip/token.png
- On-chain metadata matches: solscan.io/token/FwBixtdc…
- No duplication on Jupiter's strict list

#CHEEPEPE @cheepepe_sol

**Comment by sorena1854** [comment]
I can hereby attest that the provided information is accurate and the token is valid.
- Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- Symbol: CHEEPEPE
- Token Decimals: 6
- Token icon: https://www.cheepepe.vip/token.png
- On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- No duplication on Jupiter's strict list

**Comment by sasisasil** [comment]
I can officially confirm that the details provided are accurate, and the token remains valid.
- Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- Symbol: CHEEPEPE
- Token Decimals: 6
- Token icon: https://www.cheepepe.vip/token.png
- On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- No duplication on Jupiter's strict list

**Comment by AIleah223** [comment]
I hereby attest to the accuracy of the provided information and confirm the validity of the token.
- Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- Symbol: CHEEPEPE
- Token Decimals: 6
- Token icon: https://www.cheepepe.vip/token.png
- On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- No duplication on Jupiter's strict list

**Review by AIleah223 — COMMENTED** [review]
I confirm the accuracy of the provided information and affirm the continued validity of the token.

- Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- Symbol: CHEEPEPE
- Token Decimals: 6
- Token icon: https://www.cheepepe.vip/token.png
- On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- No duplication on Jupiter's strict list

**Comment by Suhail51998** [comment]
I can confirm that the information here is accurate and the token is valid and credible.

Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
Symbol: CHEEPEPE
Token Decimals: 6
Token icon: https://www.cheepepe.vip/token.png
On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
No duplication on Jupiter's strict list

**Review by alyssareme22 — COMMENTED** [review]
This is to certify the accuracy of the provided information and the continued validity of the token.
- Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- Symbol: CHEEPEPE
- Token Decimals: 6
- Token icon: https://www.cheepepe.vip/token.png
- On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- No duplication on Jupiter's strict list

File: validated-tokens.csv
Comment (ambutlang223) [inline validated-tokens.csv:1]: I affirm the accuracy of the provided information and confirm the continued validity of the token.
- Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- Symbol: CHEEPEPE
- Token Decimals: 6
- Token icon: https://www.cheepepe.vip/token.png
- On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
- No duplication on Jupiter's strict list

**Review by ambutlang223 — COMMENTED** [review]
I affirm the accuracy of the provided information and confirm the continued validity of the token.

Mint address: FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
Symbol: CHEEPEPE
Token Decimals: 6
Token icon: https://www.cheepepe.vip/token.png
On-chain metadata matches: https://solscan.io/token/FwBixtdcmxawRFzBNeUmzhQzaFuvv6czs5wCQuLgWWsg
No duplication on Jupiter's strict list

**Comment by Jymfletzfur** [comment]
Your image link does not match what is in metadata, please come into discord

**Comment by cheepepe** [comment]
> Your image link does not match what is in metadata, please come into discord

I have created a pull request to update it. Please check for me. Thank you very much
https://github.com/jup-ag/token-list/pull/3357

**Pull request closed without merging**